crates/state/src/exec_env.rs

```diff
@@ -19,7 +19,7 @@ pub struct ExecEnvState {
     /// seen on the corresponding DA layer yet.
     ///
     /// This must always be sorted.
-    waiting_da_blobs: Vec<da_blob::BlobSpec>,
+    waiting_da_blobs: Vec<da_blob::PayloadSpec>,
```

Comment:
This actually should be kept with the terminology and types relating to about blobs.

Reply:
How are we actually differentiating the terminologies "blob" and "payload"?
Maybe we need to rename as `waiting_da_blobs -> waiting_da_payloads`?

Reply:
I mentioned this in the comment where I proposed the renaming.

* Envelopes are a type of container that has a tag and a payload.  DA 
* DA is based around blobs identified by `BlobId`s.  DA blobs can be put into envelopes as the payload.

Reply:
I see. But what exactly is being suggested here? The [comment](https://github.com/alpenlabs/strata/pull/518#pullrequestreview-2479342341) suggests to use the term `BlobSpec` -> `PayloadSpec`.

> Maybe we need to rename waiting_da_blobs -> waiting_da_payloads?

Does this make sense then?

Reply:
In that case we were using "blob" in a broad sense that also included checkpoint data.  I'm trying to get us to be more deliberate with our language on this.